src/components/DateRangePicker.jsx

```diff
@@ -361,6 +364,8 @@ export default class DateRangePicker extends React.Component {
           onNextMonthClick={onNextMonthClick}
           monthFormat={monthFormat}
           withPortal={withPortal || withFullScreenPortal}
+          hidden={!focusedInput}
```

Comment:
Man, I should really have read @ljharb's comments more carefully before. :) `hidden` sounds good to me